devito/operations/interpolators.py

```diff
@@ -223,6 +235,14 @@ def _interp_idx(self, variables, implicit_dims=None):
                     for ((k, c), p) in zip(mapper.items(), pos)})
                     for v in variables}
 
+        # Position only replacement, not radius dependent.
+        # E.g src.inject(vp(x)*src) needs to use vp[posx] at all points
+        # not vp[posx + rx]
+        if pos_only is not None:
```

Comment:
if you turn pos_only into `pos_only=()` (instead of None) you don't need this `if` I think?

Reply:
no, the convention is to use immutable objects only, and tuple in that sense is fine ; dict/list/set aren't instead obv

for example, I'm using `=()` in ir/iet/nodes.py

alternatively, you may use `as_tuple`